tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-clear-cache-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_ORTModule_Clear_Cache
```

Comment:
ORTModule is goung to be merged within master in 2 weeks, so maybe it is better to use a name that is "orttraining" for the more general training purpose than ortmodule

Reply:
I guess this is a larger change that would require renaming more pipelines as well. I will name this job ```Onnxruntime_Linux_GPU_ORTTraining_Clear_Cache``` for now and then get to other pipelines/tasks at a later point.